pkg/apply/run.go

```diff
@@ -44,13 +45,23 @@ func NewApplierFromArgs(imageName []string, args *RunArgs) (applydrivers.Interfa
 	if err != nil && err != clusterfile.ErrClusterFileNotExists {
 		return nil, err
 	}
-	if err = cf.SetSingleMode(args.Single); err != nil {
-		return nil, err
-	}
 
 	cluster := cf.GetCluster()
 	if cluster == nil {
 		logger.Debug("creating new cluster")
+		if args.Masters == "" && args.Nodes == "" {
+			//make sure has  labring/kubernetes
+			var has bool
+			for _, s := range imageName {
+				if has = strings.Contains(s, "labring/kubernetes"); has {
```

Comment:
The basic image judgment method is incorrect, and there is no need to judge the basic image here, because the create cluster process already includes:
https://github.com/labring/sealos/blob/339adfa18ca0af11c49c33c1c5695655e74db2ca/pkg/apply/processor/interface.go#L176-L178